generation/NovelRT/Audio/include-NrtAudio.h

```diff
@@ -0,0 +1 @@
+#include <NovelRT.h>
```

Comment:
Could you clarify what these headers are with only one include line in them? The NovelRT interop headers are self-isolated and shouldn't need `NovelRT.h` unless this is your centralised header solution for interop (for now)?

Reply:
These (`include-Nrt*.h`) headers are for ClangSharp so the C bindings can be auto-generated. It simplifies the process overall and allows for any tiny fixups if they're ever required.

`NovelRT.h` is the "centralized" header and simply exposes the C API if you reference it via a C compiler or if you explicitly define `NOVELRT_C_API`. This makes it so that I don't have to include all the individual header files here and users don't need to think about what they do or don't want. Instead, they get NovelRT and the linker takes care of trimming anything unused.

Reply:
I see, sounds good to me!